.github/CODEOWNERS

```diff
@@ -81,6 +81,7 @@
 /pkg/infra/metrics/ @grafana/backend-platform
 /pkg/infra/network/ @grafana/backend-platform
 /pkg/infra/process/ @grafana/backend-platform
+/pkg/infra/proxy/ @grafana/hosted-grafana-team @grafana/backend-platform
```

Comment:
1. CODEOWNERS file is out of sync with main
2. I don't think backend-platform has anything to do with this as this is a new package, so I would suggest to keep it to hosted grafana

Reply:
👍 Merged in main and updated to be just the HG team